internal/component/azure/servicebus/client.go

```diff
@@ -174,7 +192,27 @@ func (c *Client) CloseAllSenders(log logger.Logger) {
 	close(workersCh)
 
 	// Clear the map
-	c.senders = make(map[string]*servicebus.Sender)
+	maps.Clear(c.senders)
+}
+
+// Close the client and every sender or consumer created by the connnection.
+func (c *Client) Close(log logger.Logger) {
```

Comment:
qq, back in the `GetSender` function, no context is passed to the newly created sender.
When the client is closed, does the SDK automatically close all senders or do we need to closed the senders also?

Reply:
Yes https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/messaging/azservicebus#Client.Close